Jenkinsfile

```diff
@@ -358,9 +358,6 @@ pipeline {
                       unstash 'source'
                       dir("${BASE_DIR}"){
                         powershell label: 'Install test tools', script: '.ci\\windows\\test-tools.ps1'
-                        retry(3) {
-                          bat label: 'Build', script: '.ci/windows/dotnet.bat'
```

Comment:
Instead of building it once and then running `test.bat` in a separate step we can do the whole thing in `test.bat`.